.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         uses: "actions/checkout@v2"
       - name: "Setup environment"
         uses: "networktocode/gh-action-setup-poetry-environment@v2"
+        with:
```

Comment:
why do we have to have these?

Reply:
It's not needed. I believe we discussed having the tests run on 3.11 for the speed improvements. Want me to remove them?

Reply:
What version would be it be otherwise? it's not clear to me how this is being used.

Reply:
The action [sets up](https://github.com/networktocode/gh-action-setup-poetry-environment/blob/main/action.yml) the poetry environment. It defaults to version 3.9.